### PR TITLE
Make OpenAI model configurable via environ variables

### PR DESCRIPTION
**Pull request by Cephian**
OpenAI updates models by first allowing you to to upgrade manually, then eventually upgrading the pointer from the stable version. Because of this it would be nice to be able to keep the stable version as the default value, but allow environ variable configurability for easy upgrade. See screenshot from OpenAI new model announcement email below
image

### CODE REVIEW

File: image-m13-c-183.jpg
Comment (jcogan97) [inline image-m13-c-183.jpg:1]: was this file meant to be added?

Comment (Cephian) [reply image-m13-c-183.jpg:1]: lol no idea how that got there. will remove later today

Comment (Cephian) [reply image-m13-c-183.jpg:1]: I removed it from git history

**Review by anoadragon453 — APPROVED** [review]
Time to try out GPT-4.

**Comment by Cephian** [comment]
> Time to try out GPT-4.

She's been on it for a week :^)

**Review by jcogan97 — APPROVED** [review]

**Pull request merged**